README.md

```diff
@@ -17,104 +17,139 @@ This documentation outlines the necessary and optional [dependencies](#dependenc
+- [tiff](http://www.libtiff.org/) v4.0.9: Tag Image File Format library. Downloaded automatically by conan.
+- [cfitsio](http://heasarc.gsfc.nasa.gov/fitsio/fitsio.html): v4.0.0: Library of `C` and `Fortran` subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format. Downloaded automatically by conan.
+- [yaml-cpp](https://github.com/jbeder/yaml-cpp) v0.6.3: YAML parser and emitter in `C++`. Downloaded automatically by conan.
 - [casacore](http://casacore.github.io/casacore/) - Optional - Needed to interface with measurement
-  sets. The main purify program requires this library (and its dependencies)
-- [OpenMP](http://openmp.org/wp/) v4.8.4 (Trusty) - Optional - Speeds up some of the operations.
-- [MPI](https://www.open-mpi.org) v3.1.1 (Trusty) - Optional - Parallelisation paradigm to speed up operations.
-- [spdlog](https://github.com/gabime/spdlog) v* - Optional - Logging library. Downloaded automatically if
-  absent.
-- [Catch2](https://github.com/catchorg/Catch2) v2.3.0 - Optional -  A `C++`
-  unit-testing framework only needed for testing. Downloaded automatically if absent.
-- [google/benchmark](https://github.com/google/benchmark) `v1.3.0` - Optional - A `C++`
-  micro-benchmarking framework only needed for benchmarks. Downloaded automatically if absent.
+  sets. The main **PURIFY** program requires this library (and its dependencies)
+- [spdlog](https://github.com/gabime/spdlog) v1.9.2: Optional - Logging library. Downloaded automatically by conan.
+- [Catch2](https://github.com/catchorg/Catch2) v2.13.9: Optional -  A `C++`
+  unit-testing framework only needed for testing. Downloaded automatically by conan.
+- [google/benchmark](https://github.com/google/benchmark) v1.6.0: Optional - A `C++`
+  micro-benchmarking framework only needed for benchmarks. Downloaded automatically by conan.
 
 Installing and building PURIFY
 -------------------------------------
 
-**PURIFY** can be installed through the software packet manager on Linux Debian distributions:
+To build **PURIFY**:
 
-```
-apt-get install purify
-```
+1. Once the mandatory user-provided dependencies are present, `git clone` from the [GitHub repository](https://github.com/astro-informatics/purify):
 
-Alternatively, you can build **PURIFY** entirely from the source code. Once the mandatory dependencies are present, `git clone` from the [GitHub repository](https://github.com/astro-informatics/purify):
+    ``` bash
+    git clone --recurse-submodules https://github.com/astro-informatics/purify.git
+    ```
 
-```
-git clone https://github.com/astro-informatics/purify.git
-```
+1. Create a `conan` package for `sopt`
 
-Then, the program can be built with standard `CMake` command:
+    ```bash
+    conan create /path/to/purify/sopt/ --build missing -s compiler.libcxx=libstdc++11 -pr:h=default -pr:b=default
```

Comment:
This gives an error about broken symlinks, which can be fixed by editing your `.conan/conan.conf` but did we find a way to pass that option on the command line?

Reply:
Ah, I must have had that in my conf. That's a bit annoying. You could set an [environment varible](https://docs.conan.io/en/1.46/reference/env_vars.html#conan-skip-broken-symlinks-check)

Reply:
I've added a note to the readme on how to fix this error.